Source/EventAnalysis/KTIterativeTrackClustering.hh

```diff
@@ -88,33 +97,174 @@ namespace Katydid
             MEMBERVARIABLE(double, AverageUnitlessResidualThreshold);
             MEMBERVARIABLE(unsigned, NTracks);
 
+        private:
```

Comment:
You should leave the methods declarations in the class, and move their definition outside at the bottom of the header file: this makes the class cleaner.